src/diffusers/schedulers/scheduling_ddpm.py

```diff
@@ -181,14 +182,31 @@ def scale_model_input(self, sample: torch.FloatTensor, timestep: Optional[int] =
         """
         return sample
 
-    def set_timesteps(self, num_inference_steps: int, device: Union[str, torch.device] = None):
+    def set_timesteps(
+        self,
+        num_inference_steps: int,
+        device: Union[str, torch.device] = None,
+        custom_timesteps: Optional[List[int]] = None,
```

Comment:
```suggestion
        timesteps: Optional[List[int]] = None,
```
I don't think we need to call it `custom_...` given that the method is called `set_timesteps(...)` 

Also let's make sure that `num_inference_steps` and `timesteps` cannot both be set -> in this case we should throw an error. We should also update the docstring here stating that either `num_inference_steps` or `timesteps` has to be set and explain the difference.

Reply:
Yep, agreed!